app/src/main/java/ch/epfl/sdp/drone3d/ui/mission/MappingMissionSelectionActivity.kt

```diff
@@ -132,31 +148,48 @@ class MappingMissionSelectionActivity : AppCompatActivity() {
     }
 
     private fun setupListViews() {
-        val ownerId = authService.getCurrentSession()!!.user.uid
+        /**
+         * Get the missions to show (shared if not connected, shared + private if user is connected)
+         * and make the list observe the currentListState to have the right visibility when we click
+         * on the toggle button
+         */
```

Comment:
You could move the documentation outside the function

Reply:
You are right, it was a python habit